packages/text-field/test/text-field.test.js

```diff
@@ -159,74 +99,5 @@ describe('text-field', () => {
         expect(textField.hasAttribute('focused')).to.be.true;
       });
     });
-
-    describe('autoselect', () => {
```

Comment:
Moved two tests to this suite:

https://github.com/vaadin/web-components/blob/f67d18ca4f9a38e1798a513e99402916c0d6d148/packages/field-base/test/input-control-mixin.test.js#L294